### PR TITLE
fix(publish): update peer dependencies before publish

### PR DESCRIPTION
**Pull request by MaximBalaganskiy**
Lerna ignores `peerDependencies` completely whilst they need to be in line with `devDependencies`.

Also, add missing modules to `components`

### CODE REVIEW

**Comment by bigopon** [comment]
@EisenbergEffect at the moment, dependencies are specified as peer deps. This is to combat an issue where users would have duplicated version of a modules. Though this makes lerna skip updating dependencies of packages, as it ignore peers section.

Do you see any issues with hooking into `prepublishOnly`? Description:
> prepublishOnly: Run BEFORE the package is prepared and packed, ONLY on npm publish. (See below.)

**Comment by EisenbergEffect** [comment]
No objections from me.

**Comment by bigopon** [comment]
@MaximBalaganskiy is there any other hook that let us have "what you see in git is what you get in npm"?

**Comment by MaximBalaganskiy** [comment]
`version` hook suits better - it's run right before the commit

**Comment by bigopon** [comment]
The description of `version` commands from https://docs.npmjs.com/misc/scripts. If we want t enhance this more, maybe use some more from lerna and https://docs.npmjs.com/cli/version.html
```
preversion: Run BEFORE bumping the package version.
version: Run AFTER bumping the package version, but BEFORE commit.
postversion: Run AFTER bumping the package version, and AFTER commit.
```
Looks suitable for this to me. Thanks @MaximBalaganskiy

File: packages/update-peers.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const fs = require ('fs');
+
+const data = fs.readFileSync('package.json');
+const content = JSON.parse(data.toString());
+content.peerDependencies = content.peerDependencies || {};
+for (let p of Object.getOwnPropertyNames(content.devDependencies)) {
+  if (p.startsWith('@aurelia-ux')) {
+    content.peerDependencies[p] = content.devDependencies[p];
```
Comment (bigopon) [inline packages/update-peers.js:8]: @MaximBalaganskiy are we sure that after pumping the version of a package, lerna also update the dev dependencies versions? or it waits until every package is done before bumping this? If it's the later, we may have issue here

Comment (MaximBalaganskiy) [reply packages/update-peers.js:8]: Lerna does packages one by one calling appropriate hooks. I've tried this in my local branch, works as intended.

Comment (bigopon) [reply packages/update-peers.js:8]: Ok that's awesome. Thanks for this PR. let's go then

**Pull request merged**